src/services/emails/confirm-reset-password.js

```diff
@@ -0,0 +1,29 @@
+const Promise = require('bluebird')
+const config = require('../../config/config')
+const transporter = require('../../config/nodemailer')
+
+module.exports = (user, host) => {
+  let template = {
+    from: config.EMAIL,
+    to: user.email,
+    subject: 'Style Point: Reset your password',
+    html: `
+        <p>
+            You are receiving this because you (or someone else) have requested the reset of the password for your account.
+            <br>Please click on the following link, or paste this into your browser to complete the process:<br><br>
+            http://${host}/api/reset-password/${user.resetPasswordToken}<br><br>
+            If you did not request this, please ignore this email and your password will remain unchanged.
+        </p>
+    `
+  }
+
+  return new Promise((resolve, reject) => {
```

Comment:
I don't know if you are using a Promise inside a promise because you are using Bluebird.